sim/target-agnostic.mk

```
@@ -75,16 +87,14 @@ $(VERILOG) $(HEADER) $(fame_annos): $(FIRRTL_FILE) $(ANNO_FILE) $(SCALA_BUILDTOO
 # to generate a runtime configuration that is compatible with the generated
 # hardware (BridgeModule). Useful for modelling a memory system that differs from the default.
 .PHONY: conf
-conf: $(fame_annos)
+conf: $(fame_annos) $(GEN_CLASSPATH)
```

oops, I forgot to handle converting an actual classpath into a list of dependencies here.  Should be something like 
```suggestion
conf: $(fame_annos) $(subst :, ,$(GEN_CLASSPATH))
```

Otherwise passing in an actual colon separated classpath will hit the "multiple pattern targets" error that I always have to lookup the meaning for when I hit it (see https://stackoverflow.com/questions/2100448/multiple-target-patterns-makefile-error)

---

Resolved in cd39ff3